mimic/rest/noit_api.py

```diff
@@ -0,0 +1,114 @@
+# -*- test-case-name: mimic.test.test_noit -*-
+"""
+Defines get token, impersonation
+"""
+import xmltodict
+from uuid import UUID
+from twisted.web.server import Request
+from mimic.rest.mimicapp import MimicApp
+from mimic.canned_responses.noit import (create_check, get_check,
+                                         get_all_checks, delete_check,
+                                         test_check)
+
+
+# TO DO:
```

Comment:
please file issues for these and remove the comment, the bugtracker should have all the stuff we want to do, not the code :). If you think that this would be useful for someone reading the code later, just include links to the github issues.